src/locales/ar/translation.json

```diff
@@ -1,5 +1,10 @@
 {
   "title": "[ar] Project Title",
+  "agency": {
```

Comment:
Why is this outside of the footer brace? Will it be used throughout the page?

Reply:
Yeah I used it in header too for the alt tags - it seems like something that could be used in multiple places